bokeh/client/connection.py

```diff
@@ -237,7 +237,7 @@ def request_server_info(self):
     # Private methods ---------------------------------------------------------
 
     def _formatted_url(self):
-        formatted_url = "%s?bokeh-session-id=%s" % (self._url, self._session.id)
+        formatted_url = self._url
         if self._arguments is not None:
             for key, value in self._arguments.items():
                 formatted_url += "&{}={}".format(quote_plus(str(key)), quote_plus(str(value)))
```

Comment:
@philippjfr just asking for an opinion: don't we need to make sure if there are any arguments that the first one is preceded by `?` and not `&` ?

Reply:
maybe: 
```
arguments = (f"{quote_plus(str(key))}={quote_plus(str(value))}" for key, value in self._arguments.items())
formatted_url += "?" + "&"/join(arguments)

```

Reply:
You're right, same issue as on the JS end.

Reply:
Correct if I'm wrong but I think the JS side is OK:
```
if (this.args_string != null && this.args_string.length > 0)
        versioned_url += `?${this.args_string}`
```

Reply:
It is now, but originally I had forgotten to update it and it was `&${this.args_string}`